app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Share virtual cohort component
+ * Created by Jing Su on 9/25/2017
+ */
+
+'use strict';
+(function(Vue, $, vcSession) {
+  Vue.component('shareVirtualStudy', {
+    template:
+    '<div v-if="showShareButton" class="share-virtual-study">' +
+    '<div class="share-cohort-btn">' +
+    '<i class="fa fa-share-alt" alt="Share Virtual Study"></i></div></div>',
+    props: [
+      'selectedPatientsNum', 'selectedSamplesNum',
+      'stats', 'updateStats', 'showShareButton'
+    ],
+    data: function() {
+      return {
+        savedVC: null
+      };
+    }, methods: {
+      saveCohort: function() {
+        var _self = this;
+        _self.updateStats = true;
+        _self.$nextTick(function() {
+          _self.addNewVC = true;
+        });
+      }
+    },
+    ready: function() {
+      var self_ = this;
+      var previousSelectedCases = {};
+      if (this.showShareButton) {
+        $('.share-virtual-study').qtip({
+          style: {
+            classes: 'qtip-light qtip-rounded qtip-shadow'
+          },
+          show: {event: 'mouseover', ready: false},
+          hide: {fixed: true, delay: 200, event: 'mouseleave'},
+          position: {
+            my: 'bottom center',
+            at: 'top center',
+            viewport: $(window)
+          },
+          content: 'Share Virtual Study'
+        });
+        $('.share-cohort-btn').qtip({
+          style: {
+            classes: 'qtip-light qtip-rounded qtip-shadow ' +
+            'iviz-share-cohort-btn-qtip'
+          },
+          show: {event: 'click', ready: false},
+          hide: false,
+          position: {
+            my: 'top center',
+            at: 'bottom center',
+            viewport: $(window)
+          },
+          events: {
+            render: function(event, api) {
+              var tooltip = $('.iviz-share-cohort-btn-qtip .qtip-content');
+
+              tooltip.find('.share-cohort').click(function() {
+                tooltip.find('.shared').css('display', 'none');
+                tooltip.find('.dialog').css('display', 'none');
+                tooltip.find('.saving').css('display', 'none');
+                api.reposition();
+
+                // Copy virtual study link to clipboard
+                var $temp = $("<input>");
```

Comment:
Let's not using dollar sign in variant naming. It is very confusing with other variables from jQuery and vue native function.

Reply:
Sure, I will change it.